### ISSUE TITLE
Doesn't seem to work when checkout git branch on remote server

### ISSUE THREAD

**Issue by guanw** [henrikpersson/rsub#70]
Hi, Like the title, rsub doesn't seem to recognize server git checkout changes. Is this something you plan to have in the future release? If not, I can work on this feature and probably will need some guidance from you. Thanks!

**Comment by KES777** [comment]
Currently after checkout I just reopen required files. It will be good if that will be done automatically

**Comment by guanw** [comment]
Yeah.. That's exactly what I did so far. I'm thinking about adding this functionality.
